packages/rich-text/src/RichTextEditor.mdx

```diff
@@ -61,7 +61,16 @@ import { RichTextEditor } from '@contentful/field-editor-rich-text';
     };
 
     const initialValue = JSON.parse(window.localStorage.getItem('initialValue')) || undefined
-    const [field, mitt] = useMemo(() => createFakeFieldAPI(mock => mock, initialValue), []);
+    const [fakeField, mitt] = useMemo(() => createFakeFieldAPI(mock => mock, initialValue), []);
+    // Overriding mark validation here to show subscript and superscript marks
```

Comment:
By default we don't show the superscript and subscript marks but showing in the preview so doing an override here